clients/python/play.py

```diff
@@ -1,5 +1,38 @@
+import json
+import sys
+
+from game.api import API
+from game.models import Game
+
+
 def main():
-    print("TODO")
+    team_secret = fetch_team_secret()
```

Comment:
To be honnest, I don't really care which one is the right word, but that we use 2 different words for the same thing, doesn't feel right. I understand it's not exactly the same thing, but it's really close